src/Widgets/FastView/FastList.vala

```diff
@@ -64,6 +64,11 @@ public class Noise.FastView : Gtk.TreeView {
     }
 
     construct {
+        columns = new Gee.ArrayList<Type> ();
```

Comment:
This should be in the default field.

Reply:
Well, I tried, but it looks like it just breaks everything…